### ISSUE TITLE
Internet Explorer Compatibility

### ISSUE THREAD

**Issue by erikquinn** [zlsa/atc#450]
Not sure for how long, but it appears the game is no longer drawing properly in Internet Explorer. Other than that, nothing seems to be wrong. No errors logged in console. Reported in #444.

![image](https://cloud.githubusercontent.com/assets/5103735/14559724/76629cc0-02d0-11e6-9549-97ec4d82e37a.png)

**Comment by erikquinn** [comment]
Appears to be something in `canvas.js` within 2217fcd, which was a part of the merge of #432. Looking into it now.

**Comment by erikquinn** [comment]
Okay, this was caused by IE's lack of support for the `String.fromCodePoint()` function, which is called in `canvas.js: 697,698,699`. When IE didn't know how to create the arrow character, it decides to throw in the towel and jump off a bridge to its death, rather than continue trying to draw _anything_. Crazy, I know.

I believe I have fixed the behavior by adding the `.fromCodePoint()` function to `String` during intial load. It checks to see if it has the function, and if not, adds it, so the browser will support calls to use it. With this in place, I am observing no further issues (IE 11.0.9600.18282, Chrome 49.0.2623.112, Firefox 44.0.2).
